features/admin/promotion/managing_catalog_promotions/browsing_catalog_promotions.feature

```diff
@@ -8,13 +8,13 @@ Feature: Browsing catalog promotions
         Given the store operates on a single channel in "United States"
         And there is a catalog promotion "Summer sale" with priority 50
         And there is a catalog promotion "Winter sale" with priority 100
-        And the catalog promotion "Winter sale" operates between "2021-11-10 13:45" and "2022-01-08 23:59"
+        And the catalog promotion "Winter sale" operates between "10-11-2021 13:45" and "08-01-2022 23:59"
```

Comment:
This change caused the tests for this scenario to not pass, you can revert this to an earlier version, this format has nothing to do with how we show it on the grid